init/systemd/agent_start_script.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+ETH_IFACE=enp0s2
```

Comment:
Put a comment here please why this name, and explain that it should match Qemu params

Reply:
I have added an explanation for the `ETH_IFACE` name.